programs/disks/CommandMove.cpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include "ICommand.h"
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int BAD_ARGUMENTS;
+}
+
+class CommandMove : public ICommand
+{
+public:
+    CommandMove()
+    {
+        command_name = "move";
+        command_option_description.emplace(createOptionsDescription("Help Message for move", getTerminalWidth()));
+        description = "Move file or directory from `from_path` to `to_path`\nPath should be in format './' or './path' or 'path'";
+        usage = "Usage: move [OPTION]... <FROM_PATH> <TO_PATH>";
+    }
+
+    void processOptions(
+        Poco::Util::AbstractConfiguration &,
+        po::variables_map &) const override{}
+
+    void execute_impl(
```

Comment:
```suggestion
    void executeImpl(
```
should be like this by our style